frontend/services/entities/mdm.ts

```diff
@@ -4,6 +4,15 @@ import sendRequest from "services";
 import endpoints from "utilities/endpoints";
 import { buildQueryStringFromParams } from "utilities/url";
 
+export interface IFileVaultSummaryResponse {
```

Comment:
I wonder if there is a nifty way to type the keys of this interface as FileVaultProfileStatus?

Reply:
udpated to use Record util type